[lua] Adjust morbol skills and behavior in ToAU to match retail

mug0n opened the pull request:
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Added skill usage restriction to Ameretats and Jaded Jody NM so they always use Vampiric Root whenever their target has a dispellable buff, this logic does not extend to Morbol Emperor as mentioned in #7405.
Added configuration parameters to the family mixin and configured Jaded Jody so it roams at night, unlike the regular Ameretats,
Fixed a filename typo (there is no such thing as a Greater Ameretat).

## Steps to test these changes

!pos 79.275 -20 17 51, engage an Ameretat, !tp 3000 with and without buffs on.

MowFord commented on scripts/actions/mobskills/vampiric_lash.lua:
```diff
@@ -4,12 +4,18 @@
 
 mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    -- most ToAU morbols will always use Vampiric Root when target has buffs to steal
+    if mob:getMobMod(xi.mobMod.SKILL_LIST) == 2044 and mob:getName() ~= 'Morbol_Emperor' then
```
If you want to force a mobskill based on a condition you need a mixin or otherwise put logic into the mob `onMobskillPrepare`

This only says the skill is possible to use, but that it gets preferred over other skills

mug0n replied:
Welp I should've seen `onMobWeaponSkillPrepare`. :sleeping:

mug0n committed:
added skill usage restriction for ToAU morbols, modified family mixin
to allow customization, adjusted Jaded Jody night time roaming behavior
and fixed a filename typo
